app/src/main/java/com/teamagam/gimelgimel/app/view/MainActivity.java

```diff
@@ -166,6 +173,12 @@ public void run() {
                 });
             }
         }, Message.IMAGE);
+
+        mConnectivityStatusReceiver = new ConnectivityStatusReceiver(this);
+
+        IntentFilter intentFilter = new IntentFilter(ConnectivityStatusReceiver.INTENT_NAME);
+
+        LocalBroadcastManager.getInstance(this).registerReceiver(mConnectivityStatusReceiver, intentFilter);
```

Comment:
registration should be done only on onResume .
There's double registration there. !resolve